recipes/googlesearch/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.0.3" %}
+
+package:
+  name: googlesearch
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/google/google-{{ version }}.tar.gz
+  sha256: ff7d997dbf33be666ed5deb8c5abf5d103c27ec199559b6d994923469e26bb48
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
+  host:
+    - pip
+    - python
+    - beautifulsoup4
```

Comment:
Unless you need bs4 when reunning the setup.py, you should remove it from here.

Reply:
Will try.